packages/liotapkg.sh

```diff
@@ -34,30 +34,32 @@
 liota_config="/etc/liota/liota.conf"
 package_messenger_pipe=""
 
-if [ ! -f "$liota_config" ]; then
+if [ ! -f "$liota_config" ]
+then
     echo "ERROR: Configuration file not found" >&2
     echo "You made need to copy the distributed configuration file from /usr/lib/liota/config/liota.conf to /etc/liota/liota.conf" >&2
     exit -1
 fi
 
 while read line # Read configurations from file
 do
-    if echo $line | grep -F = &>/dev/null
```

Comment:
Can this check be removed ?

Reply:
It can be removed.
I had tested.
This paragraph is just to get pkg_msg_pipe value. Without it, we still can make it well.